plugins/outputs/instrumental/instrumental.go

```diff
@@ -0,0 +1,154 @@
+package instrumental
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"github.com/influxdata/telegraf/plugins/serializers/graphite"
+)
+
+type Instrumental struct {
+	Host       string
+	ApiToken   string
+	Prefix     string
+	DataFormat string
+	Template   string
+	Timeout    int
+
+	conn net.Conn
```

Comment:
Thanks. Do I also need to check for closed connections and re-connect or is that something Telegraf will notice and re-call `Connect`?